### PR TITLE
release 4.1.0

### PR DESCRIPTION
**Pull request by Dan33l**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/v4.0.0...v4.1.0)
+
+**Implemented enhancements:**
+
+- Implement snmpv3\_user fact and snmp::snmpv3\_usm\_hash function [\#157](https://github.com/voxpupuli/puppet-snmp/pull/157) ([smoeding](https://github.com/smoeding))
+
+**Closed issues:**
+
+- Move the module to voxpupuli [\#84](https://github.com/voxpupuli/puppet-snmp/issues/84)
```
Comment (ekohl) [inline CHANGELOG.md]: This was part of the 4.0 release already. IMHO it's confusing to put it in 4.1.0 then so I'd use the `skip-changelog` label on it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- Install snmptrapd on latests ubuntu [\#168](https://github.com/voxpupuli/puppet-snmp/pull/168) ([amateo](https://github.com/amateo))
```
Comment (ekohl) [inline CHANGELOG.md]: From the commit it looks like Debian was also added. I'd modify the PR title and regenerate the changelog. Given they're listed as supported distros in metadata.json it's IMHO a bugfix as well.

File: CHANGELOG.md (hunk 1 above)
Comment (Dan33l) [reply CHANGELOG.md]: I was confused me too. I didn't know how to manage this.

So i added the `skip-changelog` tag.

File: CHANGELOG.md (hunk 2 above)
Comment (Dan33l) [reply CHANGELOG.md]: For sure it is a bugfix. I added tag `bug`

**Comment by Dan33l** [comment]
@smoeding @amateo your bugfix and enhancement are now released on the forge.

**Comment by amateo** [comment]
@Dan33l . Thank you!!

**Pull request merged**